ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -150,6 +169,7 @@ const AssetList = ({ onClickAsset, showTokensLinks }) => {
       {detectedTokens.length > 0 &&
         !isTokenDetectionInactiveOnNonMainnetSupportedNetwork && (
           <DetectedTokensBanner
+            className=""
```

Comment:
Should we make `className` not required if passing it an empty string is the solution?

Reply:
Will need to converted `detected-token-banner.js` to Typescript as well in order to do this properly. Do you think it's worth it to also handle this refactor in this PR?